libs/application/templates/health-insurance/src/fields/Review/StatusAndChildren.tsx

```diff
@@ -131,19 +145,35 @@ const StatusAndChildren: FC<ReviewFieldProps> = ({
           defaultValue={
             getValueViaPath(application.answers, 'children') as string[]
           }
+          onSelect={(value) => setChildren(value as string)}
           largeButtons={true}
           split={'1/2'}
           options={[
-            {
-              label: formatText(m.yesOptionLabel, application, formatMessage),
-              value: YES,
-            },
             {
               label: formatText(m.noOptionLabel, application, formatMessage),
               value: NO,
             },
+            {
+              label: formatText(m.yesOptionLabel, application, formatMessage),
+              value: YES,
+            },
           ]}
         />
+        {children === YES && (
+          <AlertMessage
```

Comment:
You could use the custom component that you created instead `InfoMessage` so that you don't need to duplicate the type, title and message again. :)